qa/suites/orch/cephadm/workunits/task/test_rgw_multisite.yaml

```diff
@@ -0,0 +1,43 @@
+roles:
+- - host.a
+  - mon.a
+  - mgr.a
+  - osd.0
+- - host.b
+  - mon.b
+  - mgr.b
+  - osd.1
+- - host.c
+  - mon.c
+  - osd.2
+tasks:
+- install:
+- cephadm:
+- cephadm.shell:
+    host.a:
+      - ceph mgr module enable rgw
+- rgw_module.apply:
+    specs:
+      - rgw_realm: myrealm1
+        rgw_zonegroup: myzonegroup1
+        rgw_zone: myzone1
+        placement:
+          hosts:
+           - ceph-node-0
+           - ceph-node-1
```

Comment:
```
2023-03-14T12:27:30.260 INFO:journalctl@ceph.mon.a.smithi064.stdout:Mar 14 12:27:29 smithi064 ceph-mon[94293]: osdmap e32: 3 total, 3 up, 3 in
2023-03-14T12:27:30.266 INFO:teuthology.orchestra.run.smithi064.stderr:Error EINVAL: Traceback (most recent call last):
2023-03-14T12:27:30.266 INFO:teuthology.orchestra.run.smithi064.stderr:  File "/usr/share/ceph/mgr/mgr_module.py", line 1763, in _handle_command
2023-03-14T12:27:30.266 INFO:teuthology.orchestra.run.smithi064.stderr:    return CLICommand.COMMANDS[cmd['prefix']].call(self, cmd, inbuf)
2023-03-14T12:27:30.266 INFO:teuthology.orchestra.run.smithi064.stderr:  File "/usr/share/ceph/mgr/mgr_module.py", line 462, in call
2023-03-14T12:27:30.266 INFO:teuthology.orchestra.run.smithi064.stderr:    return self.func(mgr, **kwargs)
2023-03-14T12:27:30.266 INFO:teuthology.orchestra.run.smithi064.stderr:  File "/usr/share/ceph/mgr/rgw/module.py", line 96, in wrapper
2023-03-14T12:27:30.267 INFO:teuthology.orchestra.run.smithi064.stderr:    return func(self, *args, **kwargs)
2023-03-14T12:27:30.267 INFO:teuthology.orchestra.run.smithi064.stderr:  File "/usr/share/ceph/mgr/rgw/module.py", line 189, in _cmd_rgw_realm_bootstrap
2023-03-14T12:27:30.267 INFO:teuthology.orchestra.run.smithi064.stderr:    RGWAM(self.env).realm_bootstrap(spec, start_radosgw)
2023-03-14T12:27:30.267 INFO:teuthology.orchestra.run.smithi064.stderr:  File "/lib/python3.6/site-packages/ceph/rgw/rgwam_core.py", line 559, in realm_bootstrap
2023-03-14T12:27:30.267 INFO:teuthology.orchestra.run.smithi064.stderr:    self.env.mgr.apply_rgw(rgw_spec)
2023-03-14T12:27:30.267 INFO:teuthology.orchestra.run.smithi064.stderr:  File "/usr/share/ceph/mgr/rgw/module.py", line 76, in apply_rgw
2023-03-14T12:27:30.267 INFO:teuthology.orchestra.run.smithi064.stderr:    orchestrator.raise_if_exception(completion)
2023-03-14T12:27:30.267 INFO:teuthology.orchestra.run.smithi064.stderr:  File "/usr/share/ceph/mgr/orchestrator/_interface.py", line 228, in raise_if_exception
2023-03-14T12:27:30.268 INFO:teuthology.orchestra.run.smithi064.stderr:    raise e
2023-03-14T12:27:30.268 INFO:teuthology.orchestra.run.smithi064.stderr:orchestrator._interface.OrchestratorValidationError: Cannot place <RGWSpec for service_name=rgw.myrealm1.myzone1> on ceph-node-0, ceph-node-1: Unknown hosts
2023-03-14T12:27:30.268 INFO:teuthology.orchestra.run.smithi064.stderr:
2023-03-14T12:27:30.759 INFO:journalctl@ceph.mgr.a.smithi064.stdout:Mar 14 12:27:30 smithi064 conmon[94718]: 2023-03-14T12:27:30.258+0000 7f6316d1d700 -1 log_channel(cephadm) log [ERR] : Cannot place <RGWSpec for service_name=rgw.myrealm1.myzone1> on ceph-node-0, ceph-node-1: Unknown hosts
```
@rkachach I don't think explicit hosts like this are going to work in the teuthology test

Reply:
or if we really need them, you'd have to first grab the hostnames from `ceph orch host ls` and inject them into the spec, which I think is possible but probably annoying to do

Reply:
https://pulpito.ceph.com/adking-2023-03-14_05:03:12-orch:cephadm-wip-adk-testing-2023-03-13-1654-distro-default-smithi/7206254/

Reply:
@adk3798 I removed the whole placement section as in this case it doesn't really matter where the daemons are placed.